ios/ADYRedirectComponent.h

```diff
@@ -0,0 +1,17 @@
+//
+// Copyright (c) 2023 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface ADYRedirectComponent : NSObject
+
++ (BOOL)applicationOpenURL:(nonnull NSURL *)url;
```

Comment:
The name seems a bit off. Is it something like `applicationDidOpenURL`?

Reply:
`didXXX` sounds like a delegate method.
This one is trying to process the URL from redirect

Reply:
Renamed at [ee70b87](https://github.com/Adyen/adyen-react-native/pull/121/commits/ee70b8710deaf5713e926791b16d51f94f6bcafc)